README.md

```diff
@@ -1,2 +1,153 @@
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Tomorrow <a name="about-project"></a>
+
+**Hello Tomorrow** is the first capstone project at Microverse, it's a site that describes a Global Summit which is happening in February 2023. This page is created based on design by [Cindy Shin](https://www.behance.net/adagio07).
+
+## 📘 Intro Video
+https://www.loom.com/share/a2a41a9ada0b41f68f1c6f6e6c97378b"
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+   <li>HTML</li>
+   <li>CSS</li>
+   <li>Javascript</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Will be added later</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Will be added later</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Beautifull UI**
+- **Mobile-first**
+- **Responsive**
+- **SEO and Accessibility Friendly**
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://bahirhakimy.github.io/hello-tomorrow/)
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- Getting Started -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To run the project locally, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+-Install [Git](https://git-scm.com/)
+-Install a code editor, I suggest [VsCode](https://code.visualstudio.com/)
+
+### Setup
+
+Clone the repo into your local machine:
+git clone github.com/BahirHakimy/hello-tomorrow.git
+
+### Usage
+
+You can open the `index.html` file using a web browser or using VsCode extension [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **BahirHakimi**
+
+- GitHub: [:heart:](https://github.com/bahirhakimy)
+- Twitter: [:heart:](https://twitter.com/bahirhakimy)
+- LinkedIn: [:heart:](https://linkedin.com/in/bahirhakimy)
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Selecting Tickets**
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- Contributing -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- Show your support -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project leave a start for it.
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for helping me in my journey to become a Fullstack developer 🌹
```

Comment:
- [ ] Kindly give appropriate credit to the creator of the original design. I've noticed that you mentioned this in the "Description" section of the readme, but writing it here in the "Acknowledgments" section looks more professional as this section was created for this sole purpose.

Reply:
Thanks for reminding I will update it, Thank you 😃